### ISSUE TITLE
Screen reader accessibility improvements

### ISSUE THREAD

**Issue by dannyvankooten** [ibericode/mailchimp-for-wordpress#362]

**Comment by tjlian616** [comment]
Our company's blog is targeted in an ADA lawsuit. One problem listed is that the subscription form's input field has no label with it. One solution is to add an aria-label tag to the input , like this

```
<input type="text" name="NAME" placeholder="Name..." required="" aria-lable="name">
```

Really appreciate someone can add this to this great plugin. 

Thanks!

**Comment by dannyvankooten** [comment]
Hey @tjlian616,

Agreed - I think we should add that attribute to the HTML generated by the plugin by default. I'll see to it.

Until then, please know that you can already add this attribute yourself by going into **Mailchimp for WP > Forms** and then modifying the HTML for your form fields on that page.

**Comment by tjlian616** [comment]
Thanks for the answer. I did not realize that box is editable.
